lib/api/objectCopy.js

```diff
@@ -65,6 +71,17 @@ function _prepMetadata(sourceObjMD, headers, sourceIsDestination, authInfo,
         utils.getMetaHeaders(sourceObjMD) :
         utils.getMetaHeaders(headers);
 
+    // If tagging directive is REPLACE, but you don't specify the
+    // x-amz-tagging in the request, the destination object will
+    // not have any tags.
```

Comment:
I think you mean -- "If tagging directive is REPLACE but the source object does not have tags, the destination object will not have any tags."

correct?

Is there a difference for us saving an empty object versus having undefined? When a get is done, will 0 be returned for tag count versus nothing?